administration/serializers.py

```diff
@@ -1,31 +1,10 @@
-from djoser.serializers import UserSerializer
-from django.contrib.auth import get_user_model
-from django.core.exceptions import ObjectDoesNotExist
-from rest_framework.serializers import SerializerMethodField
+from rest_framework import serializers
+from authentication.models import CustomUser
+from profiles.models import Profile
 
-User = get_user_model()
 
-
-class AdminUserSerializer(UserSerializer):
-
-    phone_number = SerializerMethodField()
-
-    def get_phone_number(self, user):
-        try:
-            return user.profile.comp_phone_number
-        except ObjectDoesNotExist:
-            return None
-
-    class Meta(UserSerializer.Meta):
-        model = User
-        fields = (
-            "id",
-            "person_name",
-            "person_surname",
-            "person_email",
-            "phone_number",
-            "last_login",
-            "is_active",
-            "is_staff"
-        )
-        read_only_fields = ("phone_number", "person_email")
+class AdminCompanySerializer(serializers.ModelSerializer):
+    person = serializers.PrimaryKeyRelatedField(queryset=CustomUser.objects.all())
```

Comment:
what field you wont to have here in person attribute?

if you want to have whole object maybe it would be better to use one more model serializer to get CustomUser?

Like in documentation: https://www.django-rest-framework.org/api-guide/relations/#nested-relationships

Reply:
Fixed